### ISSUE TITLE
Support TailwindCSS 2.0 with PostCSS 7 compatibility

### ISSUE THREAD

**Issue by Thavarshan** [pktharindu/ttall#3]
Tailwind CSS 2.0 has been out for a while and, I believe it's safe enough to upgrade from 1.x to 2.x. Besides, TailwindCSS also has a compatibility version for PostCSS 7 support so, I guess things won't break.

**Comment by Thavarshan** [comment]
Working on a PR for this. Will update when ready.

**Comment by pktharindu** [comment]
Laravel Mix v6 which supports PostCSS 8 is already out of beta.

So we could probably upgrade to Tailwind v2 along with any other dependencies we might need.

**Comment by Thavarshan** [comment]
So I ran into this problem with `@tailwindcss/ui` where node packages would not install and instead throw an error:

![Screenshot 2021-01-01 at 6 23 24 PM](https://user-images.githubusercontent.com/10804999/103438959-abeced00-4c5e-11eb-9731-8624de905d32.png)

So I did some digging and found something. Apparently, TailwindCSS v2.0 does not require `@tailwindcss/ui` [https://tailwindui.com/changes-for-v2](https://tailwindui.com/changes-for-v2).

I removed `@tailwindcss/ui`, updated the packages versions and relevant config files and that seemed to have done the trick. The update seems to be stable without any hicks. Hope it works ok.
